src/lexer.rs

```diff
@@ -236,14 +246,16 @@ impl Tokenize for Tokenizer {
                     "%=" => vec![Token::PCopy(1), Token::EMod, Token::PPop(1),
                     Token::EEq],
 
-                    "=[]" => vec![Token::EPoke(64)],
+                    "=[]" => vec![Token::EPoke(USE_DEFAULT_SIZE)],
```

Comment:
Why not using new enum value (e.g., `Token::EPokeDefaultSize`) instead of `Token::EPoke(USE_DEFAULT_SIZE)`